document-service/documents-ui/src/utils/documentRecords.ts

```diff
@@ -13,4 +13,6 @@ export function debounce(func, timeout = 200) {
       func(...args);
     }, timeout);
   };
-}
\ No newline at end of file
+}
+
+export const pageSize = 100
```

Comment:
Small thing but maybe put the const at the top of the file with a short comment describing it's purpose